internal/kms/kms.go

```diff
@@ -316,11 +316,11 @@ func GetDefaultKMS(secrets map[string]string) (EncryptionKMS, error) {
 type EncryptionKMS interface {
 	Destroy()
 
-	// RequiresDEKStore returns the DEKStoreType that is needed to be
+	// RequiresDEKStore returns the dekStoreType that is needed to be
```

Comment:
This interface is public, so the return type should be public too?

Reply:
better to be public